modules/giapi/src/main/scala/giapi/client/package.scala

```diff
@@ -156,18 +163,29 @@ package client {
       * @param url Url to connect to
       * @tparam F Effect type
       */
+    // scalastyle:off
     def giapiConnection[F[_]: Effect](url: String, commandsTimeout: Duration): GiapiConnection[F] =
       new GiapiConnection[F] {
         private def giapi(c: ActiveMQJmsProvider,
                           sg: StatusGetter,
                           cc: CommandSenderClient,
                           ss: StatusStreamer) =
           new Giapi[F] {
+            @SuppressWarnings(Array("org.wartremover.warts.Throw"))
             override def get[A: ItemGetter](statusItem: String): F[A] =
               Sync[F].delay {
                 val item = sg.getStatusItem[A](statusItem)
-                // Note item.getValue can throw if e.g. the item is unknown
-                item.getValue
+                // Note item.getValue can throw if e.g. the item is unknown, let's avoid NPEs though
+                Option(item) match {
+                  case Some(i) => i.getValue
+                  case None    => throw GiapiException(s"Status item $statusItem not found")
```

Comment:
Is an exception really necessary here?

Reply:
we discussed this over Stride and that was the conclusion. If no item is found you'll get an NPE which is maybe worse

Reply:
Sorry I merged but it is a good idea. I'll add it next